packages/core/src/components/tooltip/tooltip.scss

```diff
@@ -25,6 +19,16 @@
       margin-right: 0.35rem;
     }
   }
+
+  .tooltip-container {
```

Comment:
Container not growing with host element:

![image](https://github.com/user-attachments/assets/ad99d9ed-46bf-47e3-904f-dc51e55918a6)


```suggestion
  .tooltip-container {
      display: block;
      position: relative;
      width: 100%;
      height: 100%;
```